src/participant_network.star

```diff
@@ -129,7 +237,7 @@ def launch_participant_network(
                 )
             else:
                 ethereum_genesis_generator_image = (
-                    "ethpandaops/ethereum-genesis-generator:3.0.0-rc.18"
+                    "ethpandaops/ethereum-genesis-generator:3.0.0-rc.19"
```

Comment:
lets put this in the constants.star please!

Reply:
done